### PR TITLE
replaces gulp-metal with webpack

### PR DESCRIPTION
**Pull request by vbence86**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+const webpack = require('webpack');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+
+module.exports = {
+	entry: {
+		'Toggler.js': './src/Toggler.js',
```
Comment (jbalsas) [inline webpack.config.js]: Hey @vbence86, shouldn't this be `toggler.js` with lowercase?

File: demos/index.html [hunk 2]
```diff
@@ -5,9 +5,9 @@
 	<title>Demo: Toggler</title>
 
 	<link rel="stylesheet" href="../node_modules/bootstrap/dist/css/bootstrap.css">
-  <link rel="stylesheet" href="../build/toggler.css">
 
-	<script src="../build/globals/toggler.js"></script>
+	<link rel="stylesheet" href="../build/globals/toggler.css">
+	<script src="../build/globals/Toggler.js"></script>
```
Comment (jbalsas) [inline demos/index.html]: Used to be `toggler.js`, lowercase

**Comment by vbence86** [comment]
It's done.

**Comment by jbalsas** [comment]
LGTM! Merging

**Pull request merged**